src/events/attributes.rs

```diff
@@ -30,7 +31,85 @@ pub struct Attribute<'a> {
 }
 
 impl<'a> Attribute<'a> {
-    /// Decodes using UTF-8 then unescapes the value.
+    /// Returns the attribute value normalized as per the XML specification.
+    ///
+    /// https://www.w3.org/TR/xml/#AVNormalize
+    ///
+    /// Do not use this method with HTML attributes.
+    ///
+    /// Escape sequences such as `&gt;` are replaced with their unescaped equivalents such as `>`
+    /// and the characters \t, \r, \n are replaced with whitespace characters.
```

Comment:
```suggestion
    /// and the characters `\t`, `\r`, `\n` are replaced with whitespace characters.
```
And the same in another function. Also, the wording in the documentation are outdated, there are no more `custom_entities`, `resolve_entity` is used instead, etc.